examples/data-objects/table-document/.eslintrc.js

```diff
@@ -11,4 +11,16 @@ module.exports = {
 	rules: {
 		"@typescript-eslint/strict-boolean-expressions": "off",
 	},
+	overrides: [
+		{
+			// Rules only for test files
+			files: ["*.spec.ts", "src/test/**"],
```

Comment:
Nit. There should be no `*.spec.ts` files outside `src/test/` anyway, right? Not 100% sure about the glob syntax, if it's incorrect then just keep `src/test/**`?
```suggestion
			files: ["src/test/**/*.spec.ts"],
```

Reply:
seems like we use this glob in basically all our other packages with "test-only overrides". i'd prefer to keep that consistent over some simplification. We do want non-.spec files inside the test-folder to be exempted as well though, so if we were to change it, it should be `src/test/**`